docs/.vuepress/pr-feed-config.json

```
@@ -0,0 +1,7 @@
+{
+  "lastSelfHostedRelease": {
+    "version": "5.17.0",
+    "date": "2025-11-06",
```

should this be 2025-11-03? (5.17.0 release date)

```suggestion
    "date": "2025-11-03",
```